websocket_test.go

```diff
@@ -52,3 +52,286 @@ func TestWebsocketNil(t *testing.T) {
 
 	ws.chain.assertFailed(t)
 }
+
+func TestWebsocketExpect(t *testing.T) {
+
+	t.Run("websocket is closed", func(t *testing.T) {
+		r := newStringReporter()
+		c := &Websocket{
+			chain:    makeChain(r),
+			conn:     &websocket.Conn{},
+			isClosed: true,
+		}
+
+		c.Expect()
+
+		if !r.reported {
+			t.Errorf("Websocket.CloseWithText() error message not reported")
```

Comment:
I guess the error message is not quite correct, it should be `Websocket.Expect()` instead of `Websocket.CloseWithText()`.